include/bl_provision.h

```diff
@@ -4,21 +4,19 @@
  * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
  */
 
-#ifndef PROVISION_H_
-#define PROVISION_H_
+#ifndef BL_PROVISION_H_
+#define BL_PROVISION_H_
 
-/*
- * Copyright (c) 2018 Nordic Semiconductor ASA
- *
- * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
- */
+#ifdef __cplusplus
```

Comment:
C++-linkage guards should be after the includes

Reply:
Fixed